src/Rules/Properties/GetNonVirtualPropertyHookReadRule.php

```diff
@@ -93,6 +93,10 @@ public function processNode(Node $node, Scope $scope): array
 				continue;
 			}
 
+			if ($propertyReflection->isAbstract()->yes()) {
```

Comment:
I feel like this is a wrong fix. Instead, the foreach over the hooks above should check if the get hook has a body or not.